src/client/game/action_summary.tsx

```diff
@@ -89,6 +90,8 @@ export function ActionSummary() {
       return <ManualGoodsGrowth />;
     case Phase.DISCO_INFERNO_PRODUCTION:
       return <DiscoProduction />;
+    case Phase.POLAND_GOODS_GROWTH:
```

Comment:
Instead of building this into this file, you should instead add it to viewSettings.getActionSummary.